bundles/org.openhab.binding.livisismarthome/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,358 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="livisismarthome"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Switch Channel -->
+	<channel-type id="SwitchActuator">
+		<item-type>Switch</item-type>
+		<label>Switch</label>
+		<description>Switches the current on/off</description>
+		<category>Switch</category>
+	</channel-type>
+
+	<!-- Dimmer Channel -->
+	<channel-type id="DimmerActuator">
+		<item-type>Dimmer</item-type>
+		<label>Dimmer</label>
+		<description>Dimms the connected light</description>
+		<category>Lightbulb</category>
+	</channel-type>
+
+	<!-- RollerShutter Channel -->
+	<channel-type id="RollerShutterActuator">
+		<item-type>Rollershutter</item-type>
+		<label>Blinds Position</label>
+		<description>Controls the blinds</description>
+		<category>Blinds</category>
+		<config-description>
+			<parameter name="invert" type="boolean">
+				<label>Invert Position</label>
+				<description>When invert is true than 0 on LIVISI is UP and 100 is DOWN</description>
+			</parameter>
+		</config-description>
+	</channel-type>
+
+	<!-- BooleanSwitch Channel -->
+	<channel-type id="BooleanStateActuator">
+		<item-type>Switch</item-type>
+		<label>Switch</label>
```

Comment:
I think "Switch" is correct. It is a boolean state represented as a switch item. Therefore Switch should also be a suitable label.

Reply:
The switch is a means to change the state. So "Switch" is correct as item-type and category, but label could explain better what is being controlled and/or monitored. As an example, this is how the `system.power` channel type is defined:

```
    /**
     * Power: default system wide {@link ChannelType} which allows turning off (potentially on) a device
     */
    public static final ChannelType SYSTEM_POWER = ChannelTypeBuilder
            .state(SYSTEM_CHANNEL_TYPE_UID_POWER, "Power", CoreItemFactory.SWITCH)
            .withDescription("Device is operable when channel has state ON").withCategory("Switch")
            .withTags(List.of("Switch", "Power")).build();
```

Here the label is "Power", because this is what the switch controls.

Reply:
Ok, but it is a boolean state and the item-type is switch (on or off). ;-) So "Switch" would be a suitable label. But it's ok for me to change it to "State", I will change it.

Reply:
Is it state for an actuator (since id is "BooleanStateActuator")? In this case perhaps "Actuator State".

Reply:
solved (label renamed to "State")